public/content/developers/docs/design-and-ux/index.md

```diff
@@ -73,8 +69,7 @@ Get involved in professional community-driven organizations or join design group
 - [Deepwork.studio](https://www.deepwork.studio/)
 - [Designer-dao.xyz](https://www.designer-dao.xyz/)
 - [We3.co](https://we3.co/)
-- [Openux.xyz](https://openux.xyz/about)
-- [web3trends.org](https://web3trends.org/)
+- [Openux.xyz](https://openux.xyz/)
```

Comment:
Unsure what to do here. New site is supposed to be out in the summer, but at the moment this isnt accessible for anything. Think we can probably still leave it, but wondering what others think here.